docs/index.rst

```diff
@@ -22,12 +22,25 @@ locations.
 .. _Open edX ReadTheDocs: http://docs.edx.org/
 
 .. toctree::
-   :maxdepth: 2
+    :maxdepth: 2
+
+    best_practices
+    testing
+    javascript
+    styling
+    bootstrap
+    static_assets
+    cms_index
```

Comment:
Not sure if this is feasible, but it would be good to separate best_practice...static_assets (which are hand-written readable docs) from cms_index..openedx/modules (what are a jumble of docstrings).

Reply:
How about if I group these under "Guides" and "API Documentation" sections/pages?  Do you think I should extend the TOC to a depth of 3 in that case, or just let the sections of each page in this list drop off the main TOC?

Reply:
"Guides" and "API Documentation" is good, though "API Documentation" might sound a little optimistic.  Maybe "Python Docstrings" to set expectations?